enigma-principal/app/src/cli/app.rs

```diff
@@ -1,32 +1,67 @@
 use boot_network::{
     keys_provider_http::{PrincipalHttpServer, StateKeyRequest},
-    principal_manager::{self, PrincipalManager, ReportManager, Sampler, PrincipalConfig},
+    principal_manager::{self,
+                        PrincipalManager,
+                        ReportManager,
+                        Sampler,
+                        PrincipalConfig,
+                        SgxEthereumSigner,
+                        PrivateKeyEthereumSigner},
```

Comment:
Please stop formatting things this way...

Reply:
instead do this:
```rust
principal_manager::{
    self,
    PrincipalManager,
    ReportManager,
    Sampler,
    PrincipalConfig,
    SgxEthereumSigner,
    PrivateKeyEthereumSigner,
},
```

Reply:
I have no problem fixing it, but these are not things that are in the scope of this PR. When you're too aggressive with these formatting changes you end up with PRs that say "added local signing", and instead of being able to see the core code that was changed you end up with 25 files changed with random spacing stuff.